daffodil-test/src/test/scala/org/apache/daffodil/section17/calc_value_properties/TestComputedLengthFields.scala

```diff
@@ -43,4 +43,9 @@ class TestComputedLengthFields {
 
   @Test def test_computedLengthAroundFixedLengths1(): Unit = { runner.runOneTest("computedLengthAroundFixedLengths1") }
 
+  // DAFFODIL-2626 circular deadlock
+  // Reproduces one of the circular issues - with prefixed length for the root element surrounding
+  // text, where the alignment region
```

Comment:
Comment ends with an incomplete phrase - did you mean "where the alignment region isn't optimized out"?

Reply:
oooo. I did that on purpose just to keep you guys all on your toes :-)